src/app/shared/dialogs/dialogs-list.component.html

```diff
@@ -30,6 +30,11 @@
   [pageSizeOptions]="[5, 10, 20, 50, 100, 200]">
 </mat-paginator>
 <mat-dialog-actions>
+  <button mat-raised-button
+          [matTooltip]=selections()
```

Comment:
This needs to have quotes.  Also we should not have a function assigned to this as the Angular change detection will just run this function every time.  You should have a string on the component which is updated when the selection changes.